offline-virtual-machine.md

```diff
@@ -39,36 +104,35 @@ kubectl describe offlinevirtualmachine myvm
 # Look at the now created VirtualMachine status and associated events:
 kubectl describe virtualmachine myvm
 
-# Stop an OfflineVirtualMachine:
-kubectl patch offlinevirtualmachine myvm -p \
+# Stop the VirtualMachine created by this OfflineVirtualMachine:
+kubectl patch offlinevirtualmachine myvm --type merge -p \
     '{"spec":{"running":"false"}}'
 
-# Implicit cascade delete (first deletes the vm and then the ovm)
+# Implicit cascade delete (first deletes the VirtualMachine and then the OfflineVirtualMachine)
 kubectl delete offlinevirtualmachine myvm
 
-# Explicit cascade delete (first deletes the vm and then the ovm)
+# Explicit cascade delete (first deletes the VirtualMAchine and then the OfflineVirtualMachine)
 kubectl delete offlinevirtualmachine myvm --cascade=true
 
-# Orphan delete (The running vm is only detached, not deleted)
-# Recreating the ovm would lead to the adoption of the vm
+# Orphan delete (The running VirtualMachine is only detached, not deleted)
+# Recreating the OfflineVirtualMachine would lead to the adoption of the VirtualMachine
 kubectl delete offlinevirtualmachine myvm --cascade=false
 ```
 
-### REST API
+## What it does
 
-Third party apps can utilize the Kubernetes REST API to manipulate the
+The OfflineVirtualMachine creates VirtualMachine once the `spec.running` is set
+to `true`. The newly created VirtualMachine occupies the same namespace as the
+mother OfflineVirtualMachine. It also has the same `metadata.name` as the
 OfflineVirtualMachine.
 
-The REST API copies the structure of the Kubernetes, so it can be accessed as
-
-```text
-POST /apis/kubevirt.io/v1alpha1/namespaces/{namespace}/offlinevirtualmachine
-GET /apis/kubevirt.io/v1alpha1/namespaces/{namespace}/offlinevirtualmachine
-GET /apis/kubevirt.io/v1alpha1/namespaces/{namespace}/offlinevirtualmachine/{name}
-DELETE /apis/kubevirt.io/v1alpha1/namespaces/{namespace}/offlinevirtualmachine/{name}
-PUT /apis/kubevirt.io/v1alpha1/namespaces/{namespace}/offlinevirtualmachine/{name}
-PATCH /apis/kubevirt.io/v1alpha1/namespaces/{namespace}/offlinevirtualmachine/{name}
-```
+When set `spec.running = true` the OfflineVirtualMachine will try to keep the
+VirtualMachine running. This means: If the VirtualMachine is turned off from
+within the guest or fails. It will recreate it and start it again. The only
+way to turn the VirtualMachine off is by setting `spec.running = false`.
 
-### The deletion of objects
-WIP
+OfflineVirtualMachine also propagates the VirtualMachine spec stored in
```

Comment:
"The OfflineVirtualMachine template works just like Pod templates: Like a cookie cutter. This implies that the spec is only used for creating the new VM. Once a VM was created, then it's spec will not be changed."

Or something along those lines.

The first sentence somehow looks a little like it could be _propgagted_ at runtime.